pipeline/data_page/get_all_data_availability.py

```diff
@@ -413,39 +422,23 @@ def get_paralogs_summary(tc, depmap_paralogs_taiga_id):
     return paralogs_summary
 
 
-def get_long_reads_summary(gcloud_storage_client, depmap_long_reads_gcloud_loc):
+def get_long_reads_summary(tc, taiga_ids):
     """
     Get a summary of the long reads data available for each cell line.
 
-    Args:
-        gcloud_storage_client (storage.Client): gcloud storage client
-        depmap_long_reads_gcloud_loc (dict): gcloud location of the long reads data
-
     Returns:
-        pd.DataFrame: Indexed by ACHID or ModelID, with a column of True values that will be filterd against the Model table in future
+        pd.DataFrame: Indexed by ModelID, with a column of True values that will be filterd against the Model table in future
     """
 
     print("getting long_reads_summary...")
 
-    bucket_name = depmap_long_reads_gcloud_loc["bucket_name"]
-    prefix = depmap_long_reads_gcloud_loc["prefix"]
-    file_names = depmap_long_reads_gcloud_loc["file_names"].replace(" ", "").split(",")
-
-    # We have at least one file to process
-    assert len(file_names) > 0, "No file names provided in gcloud location"
-
-    bucket = gcloud_storage_client.bucket(bucket_name)
     unique_model_ids = set()
 
-    for file_name in file_names:
-        blob_name = f"{prefix}/{file_name}"
-        print(f"blob_name: {blob_name}")
-        blob = bucket.blob(blob_name)
-        content = blob.download_as_string()
-        df = pd.read_csv(pd.io.common.BytesIO(content), usecols=[0])
-        # Currently, the column name is "ACHID". Could be changed in the future to "ModelID" or something else?
-        assert "ACHID" in df.columns, f"Column 'ACHID' not found in file {file_name}"
-        unique_model_ids.update(df["ACHID"].unique())
+    for taiga_id in taiga_ids:
+        df = tc.get(taiga_id)
```

Comment:
@snwessel If the datasets do need to use `preprocess_omics_dataframe`, it should be used here: `preprocess_omics_dataframe(df, taiga_id)`